Manito/Manito/Screens/CreateRoom/Component/CheckRoomView.swift

```diff
@@ -15,6 +15,16 @@ class CheckRoomView: UIView {
             dateLabel.text = newValue
         }
     }
+    var name: String = "" {
+        willSet {
+            nameLabel.text = newValue
+        }
+    }
+    var person: Int = 0 {
+        willSet {
+            personLabel.text = "\(String(newValue))인"
```

Comment:
p5;

```suggestion
            personLabel.text = "\(newValue.description)인"
```
요렇게 써도 원하는대로 값이 나올겁니당

Reply:
[9fc552f](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/173/commits/9fc552f0a27c147b1f33623cc8f4275722e9e209)
짠